examples/actix-example.rs

```diff
@@ -43,32 +43,40 @@ struct Config {
 
 #[derive(Serialize, Debug)]
 #[serde(tag = "kind")]
-enum EventType<'a> {
+enum EventType {
     #[serde(rename = "hello")]
     Hello,
     #[serde(rename = "event")]
-    Event { event: Event<'a> },
+    Event { event: Event },
     #[serde(rename = "comment")]
     Comment { comment: String },
     #[serde(rename = "error")]
     Error { error: String },
 }
 
+impl From<es::Event> for EventType {
+    fn from(event: es::Event) -> Self {
+        Self::Event {
+            event: Event {
+                event_type: event.event_type,
+                data: String::from_utf8(event.data.to_vec()).unwrap(),
+                id: String::from_utf8(event.id.to_vec()).unwrap(),
+            },
+        }
+    }
+}
+
 #[derive(Serialize, Debug)]
-struct Event<'a> {
+struct Event {
     #[serde(rename = "type")]
     event_type: String,
-    data: Option<String>,
-    id: Option<&'a [u8]>,
+    data: String,
+    id: String,
 }
 
 async fn status() -> impl Responder {
     web::Json(Status {
-        capabilities: vec![
-            // "headers".into(),
-            "last-event-id".into(),
-            "read-timeout".into(),
-        ],
+        capabilities: vec![],
```

Comment:
Decided to remove all optional capabilities while we developed to make it easier to test stuff that should be working. We opt back in to one of these later.